flask_dropzone/__init__.py

```diff
@@ -26,7 +26,7 @@
 class _Dropzone(object):
 
     @staticmethod
-    def load(js_url='', css_url='', version='5.2.0'):
+    def load(js_url='', css_url='', version='latest'):
```

Comment:
I think it's may not a good idea to use `latest` as the default value since we provide built-in resources with a fixed version. We have to sync the local resource inside the extension to the latest version. Otherwise, the user may meet inconsistent results when disable/enable the `DROPZONE_SERVE_LOCAL` config.

Reply:
Yes, I also think of this after submit the PR, I will drop the code relevant.